alert-database/src/test/java/com/synopsys/integration/alert/database/api/StaticJobAccessorTest.java

```diff
@@ -315,6 +321,154 @@ void getMatchingEnabledJobsWithFrequencyTest() {
         assertEquals(jobName, distributionJobModel.getName());
     }
 
+    @Test
```

Comment:
It looks like all of these new tests have a lot of duplication between them, we can probably cut down on that duplication.

Reply:
the process for each is pretty similar but each one of them is using its channel specific model's and entities. Theres a few places I think that can be consolidated like 2 of the Mockito.when's or the assertEquals. Would you prefer those moved out into other private methods?

Reply:
If we're trying to avoid using inheritance in tests, then this should be fine and is expected